R/data.R

```diff
@@ -67,8 +77,13 @@ write_stan_json <- function(data, file) {
           is.data.frame(var) || is.list(var))) {
       stop("Variable '", var_name, "' is of invalid type.", call. = FALSE)
     }
+    if (anyNA(var)) {
+      stop("Variable '", var_name, "' has NA values.", call. = FALSE)
+    }
```

Comment:
i moved this check from `process_data` to `write_stan_json` since it should be checked even if `write_stan_json` is called directly and not via `process_data`